{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/views.py

```diff
@@ -27,6 +29,11 @@ def get_success_url(self):
     def get_object(self):
         return User.objects.get(username=self.request.user.username)
 
+    def form_valid(self, form):
+        form.save()
```

Comment:
We don't need to save the form, I think, the [base view is doing it](https://ccbv.co.uk/projects/Django/2.2/django.views.generic.edit/UpdateView/#form_valid) for us when we call `super().form_valid(form)`.